com.playeveryware.eos/Runtime/Core/Config/Attributes/ConfigFieldType.cs

```diff
@@ -78,6 +78,13 @@ public enum ConfigFieldType
         /// arbitrary functionality within a ConfigEditor.
         /// </summary>
         Button,
+
+        /// <summary>
+        /// An enum with one required value.
+        /// This will render as a dropdown list, where exactly one value can be 
+        /// selected.
+        /// </summary>
+        Enum
```

Comment:
Until the `EnumConfigFieldAttribute` class is implemented, I don't think that this should be included here.